docs/ref/django-admin.txt

```diff
@@ -364,6 +364,21 @@ standard output.
 When this option is set and ``--verbosity`` is greater than 0 (the default), a
 progress bar is shown in the terminal.
 
+Fixtures compression
+~~~~~~~~~~~~~~~~~~~~
+
+Fixtures may be compressed in ``gz``, ``bz2``, ``lzma``, or ``xz`` format.
+For example::
+
+    django-admin dumpdata -o mydata.json.gz
+
+would save the serialized data compressed into a GZip archive.
+
+.. versionchanged:: 3.2
+
+    Support for fixtures compression was added.
+
+
```

Comment:
We can chop `versionchanged` and add `versionadded`:
```
Fixtures compression
~~~~~~~~~~~~~~~~~~~~

.. versionadded:: 3.2

Fixtures ...
```

Reply:
Fixed